scripts/battlefields/Mine_Shaft_2716/bionic_bug.lua

```diff
@@ -0,0 +1,57 @@
+-----------------------------------
+-- Bionic Bug
+-- Level 75 ENM
+-- !addkeyitem SHAFT_2716_OPERATING_LEVER
+-----------------------------------
+local mineShaft2716ID = zones[xi.zone.MINE_SHAFT_2716]
+-----------------------------------
+
+local content = Battlefield:new({
+    zoneId           = xi.zone.MINE_SHAFT_2716,
+    battlefieldId    = xi.battlefield.id.BIONIC_BUG,
+    maxPlayers       = 18,
+    levelCap         = 75,
+    timeLimit        = utils.minutes(30),
+    index            = 2,
+    entryNpc         = '_0d0',
+    exitNpcs         = { '_0d1', '_0d2', '_0d3' },
+    requiredKeyItems = { xi.ki.SHAFT_2716_OPERATING_LEVER, message = mineShaft2716ID.text.SNAPS_IN_TWO, },
+    grantXP          = 3000,
+})
+
+content:addEssentialMobs({ 'Bugboy' })
+
+    content.loot =
```

Comment:
```suggestion
content.loot =
```
Not sure why LLS didnt trip on this?

Reply:
Fixed